pkg/serverless/proc/proc.go

```diff
@@ -114,3 +121,36 @@ func GetCPUData(path string) (float64, float64, error) {
 
 	return userCPUTimeMs, systemCPUTimeMs, nil
 }
+
+type NetworkData struct {
+	RxBytes float64
+	TxBytes float64
+}
+
+// GetNetworkData collects bytes sent and received by the function
+func GetNetworkData(path string) (*NetworkData, error) {
+	file, err := os.Open(path)
+	if err != nil {
+		return nil, err
+	}
+	defer file.Close()
+
+	var interfaceName string
+	var rxBytes, rxPackets, rxErrs, rxDrop, rxFifo, rxFrame, rxCompressed, rxMulticast, txBytes,
+		txPackets, txErrs, txDrop, txFifo, txColls, txCarrier, txCompressed float64
+	for {
+		_, err = fmt.Fscanln(file, &interfaceName, &rxBytes, &rxPackets, &rxErrs, &rxDrop, &rxFifo, &rxFrame,
+			&rxCompressed, &rxMulticast, &txBytes, &txPackets, &txErrs, &txDrop, &txFifo, &txColls, &txCarrier,
+			&txCompressed)
+		if errors.Is(err, io.EOF) {
+			return nil, err
+		}
+		if err == nil && strings.HasPrefix(interfaceName, lambdaNetworkInterface) {
```

Comment:
Might be helpful to log the error here if there is one. For debugging reasons.

Reply:
Oh, I think I see. If the line in the file does not match the format we're expecting, it will return an error. An error like that isn't very helpful to log, so we're just ignoring it. 👍🏽

Reply:
Yep, we ignore any lines that don't fit what we're looking for